README.md

```diff
@@ -115,6 +115,18 @@ Executing tests that do not pattern match "System"
 npm test -- --ver=alpha --invert --grep="System"
 ```
 
+## Test with core
```

Comment:
Let's document how to run the tests with the custom core and how to specify core location on the test command